tools/serverpod_cli/bin/util/prompt_to_update.dart

```diff
@@ -85,6 +63,27 @@ Future<void> updateCLI(
   }
 }
 
+YamlMap? _getcurrentPackage() {
```

Comment:
Have you tested it? Cause I'm not sure how this will work with `pubspec.yaml` I mean which yaml is it going to pick up?

Reply:
Of course, I tested it. It works well. You can easily test it by running bin/serverpod_cli.dart in debug mode.

Reply:
Also, you might be missing that it has a check `if (contents is! YamlMap) return null;`. That will help to prevent issues with the wrong type.